pandas/tests/indexes/test_base.py

```diff
@@ -1096,31 +1091,37 @@ def test_map_tseries_indices_accsr_return_index(self):
             lambda values, index: pd.Series(values, index),
         ],
     )
-    def test_map_dictlike(self, mapper):
+    def test_map_dictlike_simple(self, mapper):
         # GH 12756
         expected = Index(["foo", "bar", "baz"])
         index = tm.makeIntIndex(3)
         result = index.map(mapper(expected.values, index))
         tm.assert_index_equal(result, expected)
 
-        # TODO: replace with fixture
-        for name in self.indices.keys():
-            if name == "catIndex":
-                # Tested in test_categorical
-                continue
-            elif name == "repeats":
-                # Cannot map duplicated index
-                continue
-
-            index = self.indices[name]
-            expected = Index(np.arange(len(index), 0, -1))
-
+    @pytest.mark.parametrize(
```

Comment:
rather than duplicating the parameterisation, is it worth making mapper a fixture?